Doc/tools/templates/download.html

```diff
@@ -11,58 +11,58 @@
 {% endif %}
 
 {% block body %}
-<h1>Download Python {{ release }} Documentation</h1>
+<h1>{% trans %}Download Python {{ release }} Documentation{% endtrans %}</h1>
 
-{% if last_updated %}<p><b>Last updated on: {{ last_updated }}.</b></p>{% endif %}
+{% if last_updated %}<p><b>{% trans %}Last updated on: {{ last_updated }}.{% endtrans %}</b></p>{% endif %}
 
-<p>To download an archive containing all the documents for this version of
-Python in one of various formats, follow one of links in this table.</p>
+<p>{% trans %}To download an archive containing all the documents for this version of
+Python in one of various formats, follow one of links in this table.{% endtrans %}</p>
 
 <table class="docutils">
-  <tr><th>Format</th><th>Packed as .zip</th><th>Packed as .tar.bz2</th></tr>
-  <tr><td>PDF (US-Letter paper size)</td>
-    <td><a href="{{ dlbase }}/python-{{ release }}-docs-pdf-letter.zip">Download</a> (ca. 17 MiB)</td>
-    <td><a href="{{ dlbase }}/python-{{ release }}-docs-pdf-letter.tar.bz2">Download</a> (ca. 17 MiB)</td>
+  <tr><th>{% trans %}Format{% endtrans %}</th><th>{% trans %}Packed as .zip{% endtrans %}</th><th>{% trans %}Packed as .tar.bz2{% endtrans %}</th></tr>
+  <tr><td>{% trans %}PDF (US-Letter paper size){% endtrans %}</td>
+    <td>{% trans %}<a href="{{ dlbase }}/python-{{ release }}-docs-pdf-letter.zip">Download</a> (ca. 17 MiB){% endtrans %}</td>
```

Comment:
The numbers here will change from time-to-time (they were updated this month).

Can we add them as some sort of format string, so the digits aren't part of the string to translate, or split it up somehow?

Reply:
Yes. As https://github.com/python/cpython/pull/121454#issuecomment-2232519313 said before, the size change after text compression is very small, and the last update was python2. Still, it's good suggestion

Reply:
Some kind of format string would be perfect, but I don't know how. Splitting could be a bit confusing for the translator to understand the context.

Reply:
Something like this should work:

```jinja
{% trans download_size="17" %}<a href="{{ dlbase }}/python-{{ release }}-docs-pdf-letter.zip">Download</a> (ca. {{download_size}} MiB){% endtrans %}
```

Reply:
@tomasr8 Thanks, it indeed works. Is it possible to do something like that with `_()` ? I find {% trans %} a bit noisy.

Reply:
`{% trans %}` is typically what you should use. Maybe you could break it up into multiple lines with proper indentation to make it a bit easier to read?

Reply:
@tomasr8 Can you please check latest commit ([4b348e9](https://github.com/python/cpython/pull/122385/commits/4b348e98227c86e7a3b20e28206ca50461c572b9)) if that's what you meant by multiple lines with proper indentation ?

Reply:
It's good :) Some of the longer could be broken up a bit more but I don't know what the convention is regarding the line length for the docs